files/run_worker.sh

```diff
@@ -15,8 +15,8 @@ source /usr/bin/setup_env_in_openshift.sh
 mkdir -p "${PACKIT_HOME}/.ssh"
 chmod 0700 "${PACKIT_HOME}/.ssh"
 pushd "${PACKIT_HOME}/.ssh"
-install -m 0400 /packit-ssh/id_rsa .
-install -m 0400 /packit-ssh/id_rsa.pub .
+install -m 0400 /packit-ssh/id_ed25519 .
+install -m 0400 /packit-ssh/id_ed25519.pub .
```

Comment:
Looks like wildcards work as well with `install`

```suggestion
install -m 0400 /packit-ssh/id_* .
```